test/core/CMakeLists.txt

```diff
@@ -11,6 +11,6 @@ target_link_libraries(${target}
     hyped_core
     hyped_utils
     Eigen3::Eigen
-    Boost::boost
+    # Boost::boost
```

Comment:
?

Reply:
Done so that low power current sensor debug code could build for BBB as Boost cross-compiling wasn't working at this time
All LPC sensor debug code has now (hopefully) been removed from this PR